package.json

```diff
@@ -74,14 +76,19 @@
     "express": "^4.17.1",
     "express-winston": "^4.1.0",
     "fs-extra": "^10.0.0",
+    "http-status-codes": "2.2.0",
+    "immutable": "^4.2.4",
     "js-yaml": "^4.1.0",
+    "kujira.js": "https://github.com/Team-Kujira/kujira.js",
     "level": "^8.0.0",
     "lodash": "^4.17.21",
     "lru-cache": "^7.14.1",
     "mathjs": "^10.5.0",
     "minimist": "^1.2.6",
     "morgan": "^1.10.0",
     "near-api-js": "1.0.0",
+    "node-ts-cache": "4.4.0",
```

Comment:
Gateway is supposed to be stateless middleware that the Hummingbot client uses to send transactions - this is why Hummingbot has order tracking support. Why are you using a cache?

Reply:
Gateway still remains stateless, cache was introduced to avoid loading some heavy data all over again every time they are requested. For example, when a user fetch information from a market, since it comes from a json file or an external url and some transformation happens behind the scenes, we cache the markets for immediate use and quick responses.

This is also configurable by the user, if he/she wants to disable it they can use this option in the kujira.yml

```
cache:
  marketsData: 3600 # in seconds
  markets: 3600 # in seconds
```

We tried to make our `kujira.yml` very simple and intuitive to use. All connectors have their own world and then request different yml approaches.